scripts/unzip_backup.py

```diff
@@ -55,13 +55,20 @@ def get_history_file():
 def push_csv_to_db(extracted_csv_path):
     rowcount = 0
     manifest = json.dumps(dict())
+    _, _, files = next(os.walk(extracted_csv_path))
+    if len(files) not in (5, 9):
+        print('[ERROR] the number of log tables in uploaded report does not match preset values')
+        print('{} has length {}'.format(files, len(files)))
+        exit(1)
+    turn_of_the_month = len(files) > 5
     for bsubdir, _, csvfiles in os.walk(extracted_csv_path):
         for csvf in csvfiles:
             if csvf.endswith(".csv"):
                 csv_full_path = os.path.join(bsubdir, csvf)
                 table_name_local = os.path.splitext(csv_full_path)
-                table_name_local = table_name_local[0][-1]
-
+                table_name_local = int(table_name_local[0][-1])
+                if turn_of_the_month:
+                    table_name_local //= 2
```

Comment:
For those less familiar with the codebase, is there any value in adding a comment as to why we're performing this operation?

Reply:
Logging tables are numbered [0,3] or [0,7]. In the case of 8 logging tables, [0,2,4,6] and [1,3,5,7] are 2 set of complete reports. So I divided the number by 2 to map index.

0 -> namespace log
1 -> node log
2 -> cpu/memory log
3 -> storage log